app/src/main/java/com/swent/suddenbump/model/user/UserViewModel.kt

```diff
@@ -164,7 +164,8 @@ open class UserViewModel(
           val preferencesManager = SharedPreferencesManager(appContext)
           val workerScheduler = WorkerScheduler(appContext)
           val userRepository =
-              UserRepositoryFirestore(Firebase.firestore, preferencesManager, workerScheduler)
+              UserRepositoryFirestore(
+                  Firebase.firestore, preferencesManager, workerScheduler, context)
```

Comment:
bad practice to pass the context to repository firestore directly.
the context is available inside of worker scheduler and shared preferences manager. you can pass one of them down to image repository firestore and then put a getcontext function inside the class you passed down so that image repository firestore can have access to the context.
please change this!